test/cluster/test_tablets_merge.py

```
@@ -640,3 +640,111 @@ async def finished_merge():
+
+        # Disable auto-compaction via API (not via strategy 'enabled=false'
+        # which replaces ICS with NullCompactionStrategy).
+        await manager.api.disable_autocompaction(server.ip_addr, ks, 'test')
+
+        n_keys = 5000
+        blob_value = 'x' * 4096
+
+        # Step 1: Write live data, flush.
+        batch_size = n_keys // 10
+        for i in range(0, n_keys, batch_size):
+            await asyncio.gather(*[cql.run_async(f"INSERT INTO {ks}.test (pk, c) VALUES ({k}, '{blob_value}')")
```

Ruff (0.16.1)

[error] Possible SQL injection vector through string-based query construction

(S608)